components/src/PaymentForm/MockHttpResponses/GET/paymentInstrumentForm_response.json

```diff
@@ -0,0 +1,746 @@
+{
+	"self": {
+		"type": "elasticpath.collections.links",
+		"uri": "/?zoom=defaultcart:order:paymentmethodinfo:element:paymentinstrumentform,defaultprofile:paymentmethods:element:paymentinstrumentform",
+		"href": "http://10.11.12.117:8080/cortex/?zoom=defaultcart:order:paymentmethodinfo:element:paymentinstrumentform,defaultprofile:paymentmethods:element:paymentinstrumentform"
```

Comment:
This is the internal dev environment. We'll need to update the url to point to the public instance `https://reference.epdemos.com`

Reply:
WIP